testing-pre-releases.md

```diff
@@ -0,0 +1,530 @@
+  the .deb or .rpm packages. e.g. Use v1.10.3 GA packages for kubelet, kubectl, but test a local build of kubernetes.
+- The `--kubernetes-version` flag of kubeadm or the `kubernetesVersion` field of the [kubeadm config file](https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-init),
+  defining the version number of the control plane components installed by kubeadm.
+
+For each of the above set version number, it is also necessary to understand the supported version skew policy,
+the availability of pre-compiled artifacts, or, as alternative, how to create a local version through the build process.
+
+Once all the required pieces will be available, it is possible to create a Kubernetes cluster with kubeadm and
+execute tests.
+
+<!-- TOC -->
+
+- [Testing pre-release versions of Kubernetes with kubeadm](#testing-pre-release-versions-of-kubernetes-with-kubeadm)
+    - [Overview](#overview)
+    - [Kubeadm version skew policy](#kubeadm-version-skew-policy)
+    - [Availability of pre-compiled release artifacts](#availability-of-pre-compiled-release-artifacts)
+        - [Getting .deb or .rpm packages form repository](#getting-deb-or-rpm-packages-form-repository)
+        - [Getting .deb or .rpm packages form a GCS bucket](#getting-deb-or-rpm-packages-form-a-gcs-bucket)
+        - [Getting kubeadm binaries from a GCS bucket](#getting-kubeadm-binaries-from-a-gcs-bucket)
+        - [Getting docker images from a GCR registry](#getting-docker-images-from-a-gcr-registry)
+    - [Create a local version](#create-a-local-version)
+        - [Build .debs packages](#build-debs-packages)
+        - [Build kubeadm binary](#build-kubeadm-binary)
+        - [Build controlplane docker images](#build-controlplane-docker-images)
+    - [Creating the Kubernetes cluster with kubeadm](#creating-the-kubernetes-cluster-with-kubeadm)
+    - [Testing the Kubernetes cluster](#testing-the-kubernetes-cluster)
+        - [Smoke test](#smoke-test)
+            - [DNS](#dns)
+            - [Deployments](#deployments)
+            - [Port Forwarding](#port-forwarding)
+            - [Logs](#logs)
+            - [Exec](#exec)
+            - [Services](#services)
+        - [Conformance test](#conformance-test)
+    - [Tips and Tricks](#tips-and-tricks)
+        - [Semantic version ordering](#semantic-version-ordering)
+        - [How to identify exact version/build number for a PR](#how-to-identify-exact-versionbuild-number-for-a-pr)
+        - [Change the target version number when building a local release](#change-the-target-version-number-when-building-a-local-release)
+
+<!-- /TOC -->
+
+## Kubeadm version skew policy
+
+The [Kubeadm version skew policy](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/#version-skew-policy)
+can briefly be summarized in:
+
+- The kubeadm CLI tool of version MAJOR.MINOR may deploy clusters with a control plane of version MAJOR.MINOR or MAJOR.(MINOR-1).
```

Comment:
Would it be worth mentioning the PATCH version as part of this?

Reply:
@liztio @detiber I'm going to remove the details of the version skew here and leave only the links to "official" version skew policy in Kubernestes.io.
If something should be improved or fixed, let's do this on the master copy